### ISSUE TITLE
Support zipped datapackage file

### ISSUE THREAD

**Issue by danielhuppmann** [rgieseke/pandas-datapackage-reader#13]
When trying to use this package in https://github.com/IAMconsortium/pyam/pull/323, it seemed like I needed to unpack a zipped file first (compared to just reading it as `package = Package(path)` using [datapackage-py](https://github.com/frictionlessdata/datapackage-py)).

Was I doing this wrong, or is zip really not supported?

**Comment by rgieseke** [comment]
Yes, it's currently not supported as i hadn't encountered Data Packages in the wild which i wanted to read ...

Since OPSD is having them for example, it might be a good feature to add though!
Otherwise unpacking first or on the fly would be the way to go ...
